### PR TITLE
Improve README.md phrasing and added Foreman-tasks manual link

### PR DESCRIPTION
**Pull request by Bmkiel**
Greetings,

I added the hyperlink for the Foreman-tasks manual, addressed phrasing and spelling issues.

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,8 +156,8 @@ Tasks cleanup
 -------------
 
 Although, the history of tasks has an auditing value, some kinds of
-tasks can grow up in number quite soon. Therefore there is a mechanism
-how to clean the tasks, using a rake command. When running without
+tasks can rapidly increase. Therefore, there is a mechanism for
+how to clean the tasks which is done by using a rake command. When running without
```
Comment (ezr-ondrej) [inline README.md:160]:
```suggestion
cleaning up the tasks which is done by using a rake command. When running without
```

**Review by ezr-ondrej — COMMENTED** [review]
Nice @Bmkiel ! Thanks for taking a look, I think you've missed rephrasing of one sentence?
Tho I'm not native speaker, so I'm not sure whats better :thinking:

File: README.md (hunk 1 above)
Comment (Bmkiel) [reply README.md:160]: Hey @ezr-ondrej! I like your suggested change of, "cleaning up the tasks which is done by using a rake command. When running without" and think it does smooth out the sentence structure.

File: README.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ Tasks cleanup
 
 Although, the history of tasks has an auditing value, some kinds of
```
Comment (swadeley) [inline README.md:158]:
```suggestion
Although the history of tasks has an auditing value, some kinds of
```

File: README.md [hunk 3]
```diff
@@ -157,7 +157,7 @@ Tasks cleanup
...
 tasks can rapidly increase. Therefore, there is a mechanism for
-how to clean the tasks which is done by using a rake command. When running without
+cleaning up the tasks which is done by using a rake command. When running without
```
Comment (swadeley) [inline README.md:160]: it is better than it was, but you could simplify to:
```suggestion
cleaning up the tasks using a rake command. When running without
```

File: README.md [hunk 4]
```diff
@@ -179,7 +180,7 @@ override the default configuration inside the configuration
 ```
 :foreman-tasks:
   :cleanup:
-# the period after witch to delete all the tasks (by default all tasks are not being deleted after some period)
+# the period after which to delete all the tasks (by default all tasks are not being deleted after some period)
```
Comment (swadeley) [inline README.md:183]:
```suggestion
# the period after which to delete all the tasks (by default, all tasks are not deleted after some period)
```

**Review by swadeley — APPROVED** [review]
ACK

**Review by ezr-ondrej — APPROVED** [review]
Thanks @Bmkiel ! :+1:

**Pull request merged**